src/lib/util/removeNullAndUndefinedFields.ts

```diff
@@ -14,6 +14,17 @@ export function removeNullAndUndefinedFields<T extends []>(
   something: T,
 ): DeepPartial<T>;
```

Comment:
If an input variable for this function is typed, the new typing above lets TypeScript deduce the function's result type more closely.